samples/Tests/MediaAudio.json

```diff
@@ -0,0 +1,40 @@
+{
+	"$schema": "http://adaptivecards.io/schemas/adaptive-card.json",
+	"type": "AdaptiveCard",
+	"version": "1.0",
+	"body": [
+		{
+			"type": "TextBlock",
+			"text": "This is a media element which specifies a poster:",
+			"wrap": true
+		},
+		{
+			"type": "Media",
+			"poster": "http://adaptivecards.io/content/cats/1.png",
+			"sources": [
+				{
+					"mimeType": "audio/mp4",
+					"url": "https://www.mfiles.co.uk/mp3-downloads/chopin-nocturne-op9-no2.mp3"
```

Comment:
have your checked the licensing on this? just checking :)

Reply:
You're totally, right, removing this link until we have a good mp3 source

[https://www.mfiles.co.uk/licensing.htm](https://www.mfiles.co.uk/licensing.htm)



---
In reply to: [202834078](https://github.com/Microsoft/AdaptiveCards/pull/1716#discussion_r202834078) [](ancestors = 202834078)